data/mods/PKs_Rebalance/pk_critter_zombie.json

```diff
@@ -974,7 +974,7 @@
         },
         ["LUNGE", 30]],
     "description":"Its skin stretched so tight as to have split in several places, this once-human corpse bears bones jutting out at unnatural angles.",
-    "flags":["SEES", "HEARS", "STUMBLES", "HARDTOSHOOT", "REVIVES", "NO_BREATHE", "BONES", "NO_NECRO", "POISON"]
+    "flags":["SEES", "HEARS", "STUMBLES", "HARDTOSHOOT", "REVIVES", "NO_BREATHE", "BONES", "NO_NECRO"]
```

Comment:
Without the `POISON` flag (here and 2 other places), it will yield sinew and untainted bones when butchered.